src/Logic.hs

```diff
@@ -908,7 +915,10 @@ describeStatus prId pr state = case Pr.classifyPullRequest pr of
                                                   ]
   PrStatusBuildStarted url -> Text.concat ["[CI job :yellow_circle:](", url, ") started."]
   PrStatusIntegrated -> "The build succeeded."
-  PrStatusIncorrectBaseBranch -> "Merge rejected: the target branch must be the integration branch."
+  PrStatusIncorrectBaseBranch ->
+    let BaseBranch baseBranchName = Pr.baseBranch pr
+    in format "Merge rejected: the pull request's base branch ({}) must be the same as the repository's ({}}."
+              [baseBranchName, projectBaseBranchName]
```

Comment:
`({}}` -> `({})`

Also, on style: It reads easier for me if it says `the base branch of the pull request` instead of `the pull requests's base branch`
How about the following message:

```suggestion
    in format "Merge rejected: the base branch of this pull request must be set to {}. It is currently set to {}."
              [projectBaseBranchName, baseBranchName]
```